stdlib/public/stubs/CommandLine.cpp

```diff
@@ -58,6 +58,11 @@ extern "C" char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
   return *_NSGetArgv();
 }
 #elif defined(__linux__) || defined(__CYGWIN__) || defined(__FreeBSD__)
+#if defined(__FreeBSD__)
```

Comment:
Great pointer, thanks! I was never quite happy with requiring /proc to be mounted.. Hope commit ede70f2 is a better FreeBSD fit.